bigbluebutton-html5/imports/api/common/server/helpers.js

```diff
@@ -36,6 +36,19 @@ export const parseMessage = (message) => {
   return parsedMessage;
 };
 
+export const textToMarkdown = (message) => {
+  let parsedMessage = message || '';
+  parsedMessage = parsedMessage.trim();
+
+  // replace url with markdown links
+  const urlRegex = /(?<!\]\()https?:\/\/(www)?..*?\.[a-zA-Z]{1,6}/gm;
```

Comment:
This regex is cutting part of the URL (`.br`):
![image](https://github.com/bigbluebutton/bigbluebutton/assets/5660191/e52b7625-44d1-4b1c-adb4-3070c23786d6)

I suggest some tweaks:

```suggestion
  const urlRegex = /(?<!\]\()https?\:\/\/([\d\w\-]\.?)+(\w{1,6})/gm;
```

Reply:
I didn't understand what is this sequence of chars `<!](` that will make it ignore the following URL.

Reply:
Actually I think we need a more complex Regex, like:
`https?:\/\/([\w-]+\.)+\w{1,6}([\/\?\=\&\#\.]?[\w-]+)*`

Thinking about URLs like this:
https://www.google.com.tr/admin/subPage?qs1=sss1&qs2=sss2&qs3=sss3#Services
https://google.com.tr/test/subPage?qs1=sss1&qs2=sss2&qs3=sss3#Services
http://google.com/test/subPage?qs1=sss1&qs2=sss2&qs3=sss3#Services